.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python:
+  - '3.6'
+install:
+  - 'pip install pipenv'
+  - 'pip install coveralls'
+  - 'pip install coverage'
+  - 'pipenv install'
+
+services:
+  - postgresql
```

Comment:
add a postgres addon for postgres 9.6

Reply:
_incorporated the addon for postgres 9.6_